docs/pages/installation/_index.md

```diff
@@ -5,20 +5,252 @@ weight: 2
+
+##### Supported CNIs
+
+Liqo supports K8s clusters using the following CNIs:
+* [Weave](https://github.com/weaveworks/weave)
+* [Flannel](https://github.com/coreos/flannel)
+* [Canal](https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel)
+* [Calico](https://www.projectcalico.org/)
+
+##### Configuration 
+
+You only have to export the KUBECONFIG environment variable. 
+Otherwise, liqoctl will try to use the kubeconfig in kubectl default path (i.e. `${HOME}/.kube/config` )
+
+```bash
+export KUBECONFIG=/your/kubeconfig/path
+```
+
+{{% /tab %}}
+
+{{% tab name="EKS" %}}
+
+##### Supported CNIs
+
+Liqo supports EKS clusters using the default CNI:
+* [AWS EKS - amazon-vpc-cni-k8s](https://github.com/aws/amazon-vpc-cni-k8s)
+
+##### Configuration
+
+Compared to the other K8s managed providers, Liqo leverages AWS credentials to authenticate peered clusters.
+Therefore, Liqo on AWS requires some extra permissions.
+In addition to the read-only permissions used to configure the cluster installation, Liqo uses AWS users to map peering access
+to EC2 cluster.
+
+The user used for the installation should be able to:
+* **Manage users and Policies** (*"iam:CreateUser","iam:GetUser","iam:CreateAccessKey", "iam:ManageAccessKey","iam:AttachUserPolicy","iam:CreatePolicy","iam:GetPolicy",
+  "iam:GetPolicyVersion"*).
+* **Fetch details about the cluster amd its network configuration** (*"eks:DescribeCluster","ec2:DescribeVpcs", "ec2:DescribeSubnets"*).
+
+To install Liqo on EKS, you should log in using the AWS cli (if you already did that, you can skip this step)
+This is widely documented on the [official CLI documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+
+In a nutshell, after having installed the CLI, you have to set up your identity:
+```bash
+aws configure
+```
+
+Second, you should export the cluster's kubeconfig if you have not already. You may use the following CLI command:
+
+```bash
+aws eks --region ${EKS_CLUSTER_REGION} update-kubeconfig --name ${EKS_CLUSTER_NAME}
+```
+{{% /tab %}}
+
+{{% tab name="AKS" %}}
+
+##### Supported CNIs
+
+Liqo supports AKS clusters using the following CNIs:
+* [Azure AKS - Kubenet](https://docs.microsoft.com/en-us/azure/aks/configure-kubenet)
+* [Azure AKS - Azure CNI](https://docs.microsoft.com/en-us/azure/aks/configure-azure-cni)
+
+##### Configuration
+
+First, you should have the AZ cli installed and your AKS cluster deployed. If you haven't, you can follow the [official guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+Second, you should log-in:
+```bash
+az login
+```
+
+First, let's start exporting required variables:
+
+```bash
+export AZURE_RESOURCE_GROUP=my-resourceGroup # the resourceGroup where the cluster is created
+export AKS_RESOURCE_NAME=my-cluster # the name of AKS cluster resource on Azure
+export AZURE_SUBSCRIPTION_ID=subscription-id # the subscription id associated to the AKS cluster's resource group 
```

Comment:
To me, exporting those variables adds no value, as the correct values could be directly inserted in the install command below (which adds also them to the history for further use). Just personal opinion though.